I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/tracker.rpy", line 11, in script
    jump continue6
ScriptError: could not find label 'continue6'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/tracker.rpy", line 11, in script
    jump continue6
  File "C:\Users\alex1\OneDrive\Desktop\renpy-8.0.3-sdk\renpy\ast.py", line 1974, in execute
    rv = renpy.game.script.lookup(target)
  File "C:\Users\alex1\OneDrive\Desktop\renpy-8.0.3-sdk\renpy\script.py", line 927, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'continue6'.

Windows-10-10.0.19044 AMD64
Ren'Py 8.0.3.22090809
 Version 1.0
Mon Apr 10 03:28:04 2023
